### DISCUSSION TITLE
net::ERR_ABORTED 504 (Outdated Optimize Dep)

### DISCUSSION THREAD

**Discussion by venkatmiriyala19** [pmndrs/drei#2110] [Q&A]
I am building a vite project and imported react-three/drei. At first it worked fine, but now it is showing 
<img width="337" alt="{D0E2430E-D6ED-4398-BF47-B32346AD0C34}" src="https://github.com/user-attachments/assets/81c92234-689d-474b-9832-aa53e8aa5b97">

**Comment by lowpolypanda** [comment]
Hi, did you manage to fix it?

**Reply by venkatmiriyala19** [reply]
As far as I could remember - No, the error fixed itself after hosting to vercel. I mean it didn't work on my local pc but worked on vercel deploy.

**Reply by lowpolypanda** [reply]
Ok, thank you for your help :)
